package.json

```diff
@@ -25,22 +25,23 @@
     "url": "https://github.com/atais/ng2-eonasdan-datetimepicker.git"
   },
   "devDependencies": {
-    "@types/core-js": "^0.9.34",
-    "ts-loader": "^1.2.2",
-    "typescript": "^2.0.10",
+    "@types/core-js": "^0.9.36",
+    "ts-loader": "^2.0.1",
+    "typescript": "^2.1.6",
     "zone.js": "^0.7.2"
   },
   "dependencies": {
-    "@angular/common": "^2.0.0",
-    "@angular/compiler": "^2.0.0",
-    "@angular/core": "^2.0.0",
-    "@angular/platform-browser": "^2.0.0",
-    "@angular/platform-browser-dynamic": "^2.0.0",
-    "bootstrap": "*",
+    "@angular/common": "^2.4.4",
```

Comment:
Could this be left with `"^2.0.0"` ? Or At least not "^2.4.4"
As it is public library you can not rely on the newest version of angular since some people might still be limited to the older version.